### ISSUE TITLE
Natural disaster list  for each wilaya

### ISSUE THREAD

**Issue by abderrahmaneMustapha** [dzcode-io/leblad#21]
add a list of natural disaster for each Wilaya 
example :
```json
 {
 "WilayaName": {
    "natural disaster":{
        "earthquakes" : [
             {"name": "name" , "date " :  22-3-2000, "deaths" : 1 , .more ......}
         ],
        "floods": [.......] 
      ........
}
}
}
```

**Comment by Fcmam5** [comment]
Would you add this to the original dataset or add it as a plugin?  
I personally prefer having it as an external dependency, and if I want to use it I'll just install it. Internally that plugin will just merge our data array with this `natural disaster` by wilaya array. This idea could be the same for each additional information especially those who can be updated consistently (natural disasters, population...)

**Comment by abderrahmaneMustapha** [comment]
we can schedule a call and talk about it !

**Comment by Fcmam5** [comment]
As a wrap-up for our call:
- We add an `.extend` functionality where we extend our library with a new plugin containing a dataset+functions

**Comment by Redaloukil** [comment]
Yes, @Fcmam5 seems like an excellent idea. I'm really curious to see an implementation of a plugin or extension base on the dateset. and better to move this discussion to algeria api and create a proof of concept of how plugin should be implemented.If everything goes fine, we can mention the plugin in the project as a guide. what do you think ?

**Comment by abderrahmaneMustapha** [comment]
hi @Redaloukil  i already start the implementation of this plugin  [click here](https://github.com/abderrahmaneMustapha/algeria-api-natural-disaster)

**Comment by ZibanPirate** [comment]
@Fcmam5 @Redaloukil @abderrahmaneMustapha we need to work closely with [algeria-api-natural-disaster](https://github.com/abderrahmaneMustapha/algeria-api-natural-disaster)
 Will put a draft of the requirements and then As suggested by @Redaloukil we will do a PoC, once tested and proven to be stable, we merge.

Let's keep this thread for highlights and progress, will move the discussion to the [slack workspace](https://join.slack.com/t/dzcode/shared_invite/zt-ek9kscb7-m8z_~cBjX79l~uchuABPFQ)

**Comment by Fcmam5** [comment]
I'd appreciate scheduling a call to discuss this

**Comment by ZibanPirate** [comment]
@Fcmam5 sure thing, @Redaloukil it would be great if you could join an open discussion on [slack](https://join.slack.com/t/dzcode/shared_invite/zt-ek9kscb7-m8z_~cBjX79l~uchuABPFQ)
